src/test/java/org/assertj/core/api/junit/jupiter/AbstractSoftAssertionsExtensionIntegrationTests.java

```diff
@@ -68,10 +68,11 @@ final void test_instance_per_class_with_nested_tests() {
   private void assertExecutionResults(Class<?> testClass, boolean nested) {
     EngineTestKit.engine("junit-jupiter")
                  .selectors(selectClass(testClass))
+                 .configurationParameter("junit.jupiter.conditions.deactivate", "org.junit.jupiter.engine.extension.DisabledCondition")
```

Comment:
It would have been better to use `DisabledCondition.class.getName()` to avoid relying on a String value but `DisabledCondition` is not public, is making it public something that can be considered ?

This is not a blocker for this PR ;-)

Reply:
No, we have no intention to make that class public: it's an internal implementation detail of the Jupiter test engine.

But... good that you brought it up, since there is in fact no need to be that specific. What we are really trying to do here is disable the tests unless we execute them via the Test Kit. So I'll simplify that by deactivating all conditions with `"*"`. 😉